src/components/EntityInfo/EntityMenu.vue

```diff
@@ -391,29 +391,50 @@ export default class EntityMenu extends Mixins(AllowableActionsMixin) {
     return 'Amalgamate with other businesses.'
   }
 
+  /** Whether to disable the View and Change Business Information button. */
+  get isChangeBusinessInfoDisabled (): boolean {
+    // enable if business is Not In Good Standing and user isn't staff
+    // clicking button will show a dialog -- see promptChangeBusinessInfo()
+    if (!this.isGoodStanding && !this.isRoleStaff) return false
```

Comment:
Regular users are not allowed to change business info when NIGS but we still want to show them the button.